virt/virt-4-10-release-notes.adoc

```diff
@@ -72,22 +72,28 @@ The SVVP Certification applies to:
 [id="virt-4-10-storage-new"]
 === Storage
 
+//CNV-16673 New CSI driver for HPP
+* You can use the xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Kubernetes Container Storage Interface (CSI) driver] with the hostpath provisioner (HPP) feature to configure local storage for your virtual machines. By creating an HPP custom resource that includes one or more storage pools, as well as creating a storage class for the CSI driver, you can run the HPP with less disruption to your existing {product-title} nodes and clusters.
+
 [id="virt-4-10-web-new"]
 === Web console
 
-// NOTE: no deprecated features in 4.10; commenting out these sections to retain for future use
-//[id="virt-4-10-deprecated-removed"]
-//== Deprecated and removed features
+// NOTE: Comment out deprecated and removed features (and their IDs) if not used in a release
+[id="virt-4-10-deprecated-removed"]
+== Deprecated and removed features
 
-//[id="virt-4-10-deprecated"]
-//=== Deprecated features
+[id="virt-4-10-deprecated"]
+=== Deprecated features
 
 //Deprecated features are included in the current release and supported. However, they will be removed in a future release and are not recommended for new deployments.
 
+* In {VirtProductName} 4.10, the HPP Operator configures the Kubernetes Container Storage Interface (CSI) driver. The Operator also recognizes the existing (legacy) format of the custom resource. If you upgrade to {VirtProductName} 4.10, no action is required.
```

Comment:
Also see my earlier comment about combining the two release notes.

Reply:
Discussed.